src/utils/input.js

```diff
@@ -4,8 +4,12 @@ export function parseArgs(args) {
   // The following is inspired from ethereum/remix (Copyright (c) 2016)
   // https://github.com/ethereum/remix/blob/89f34fc4f35eca0c386379550c300205d35bfae0/remix-lib/src/execution/txFormat.js#L51-L53
   try {
-    args = args.replace(/(?<=(,|\[|^)\s*)(\d+)(?=\s*(,|\]|$))/g, '"$2"') // replace non quoted number by quoted number
-    args = args.replace(/(?<=(,|\[|^)\s*)(0[xX][0-9a-fA-F]+)(?=\s*(,|\]|$))/g, '"$2"') // replace non quoted hex string by quoted hex string
+    const MATCH_HEX = /(^|([,[]\s*))(0[xX][0-9a-fA-F]+)/g
+    args = args.replace(MATCH_HEX, '$2"$3"') // replace non quoted hex string by quoted hex string
+
+    const MATCH_NUMBERS = /(^|([,[]\s*))(\w+)/g
+    args = args.replace(MATCH_NUMBERS, '$2"$3"') // replace non quoted number by quoted number
```

Comment:
I like that this would now match strings as well, we are actually adding new features. Before, `foo, bar` would not pass, and now this regex should properly catch them and enclose them in quotes. Do you mind adding a test for that, and rename `MATCH_NUMBERS` to `MATCH_WORDS`?

Reply:
sure!